doc/contributing/adding-v8-fast-api.md

```diff
@@ -38,6 +38,19 @@ for example, they may not trigger garbage collection.
 * The fast callback must be idempotent up to the point where error and fallback
   conditions are checked, because otherwise executing the slow callback might
   produce visible side effects twice.
+* If the receiver is used in the callback it must be passed as a second argument
+  leaving the first one unused.
```

Comment:
```suggestion
  leaving the first one unused, to prevent the JS land from accidentally omitting the receiver when
  invoking the fast API method.
```

I am not quite sure this is a *must*, though. There are valid reasons to actually make use of the receiver. It's just that when the receiver is expected to be the binding object, it's better to make them explicit and pass them in as the first argument.